test/swar/BasicOperations.cpp

```diff
@@ -4,9 +4,192 @@
 
 #include <type_traits>
 
+namespace zoo::swar {
+
+template<int NB, typename B>
```

Comment:
It is more readable to have the typenames be at least somewhat self describing (I realize they are immediately used by by a typedef, but still)

Reply:
I use this type of exceedingly brief identifiers to indicate they are not important.